### ISSUE TITLE
 [Bug]:   mpc is broken since upgrade 

### ISSUE THREAD

**Issue by macerg** [termux/termux-packages#11396]
### Problem description

mpc doesn't connect to  mpd on my server since recent update.

### What steps will reproduce the bug?

mpc -p 6600.

### What is the expected behavior?

Should connect to mpd.
But get MPD error: No such file or directory

### System information

termux-info:

```Termux Variables:
TERMUX_API_VERSION=0.50.1
TERMUX_APK_RELEASE=F_DROID
TERMUX_APP_PACKAGE_MANAGER=apt
TERMUX_APP_PID=9045
TERMUX_IS_DEBUGGABLE_BUILD=0
TERMUX_MAIN_PACKAGE_FORMAT=debian
TERMUX_VERSION=0.118.0
Packages CPU architecture:
aarch64
Subscribed repositories:
# sources.list
deb https://packages-cf.termux.dev/apt/termux-main stable main
# root-repo (sources.list.d/root.list)
deb https://packages-cf.termux.dev/apt/termux-root root stable
# x11-repo (sources.list.d/x11.list)
deb https://packages-cf.termux.dev/apt/termux-x11 x11 main
Updatable packages:
All packages up to date
termux-tools version:
1.24
Android version:
10
Kernel build information:
Linux localhost 4.9.179-perf+ #1 SMP PREEMPT Tue Jul 13 22:40:25 CST 2021 aarch64 Android
Device manufacturer:
OnePlus
Device model:
ONEPLUS A6003


```

**Comment by Grimler91** [comment]
How/where is the server running? 

Termux's mpd creates a socket at [`$PREFIX/var/run/mpd.socket` per default](https://github.com/termux/termux-packages/blob/master/packages/mpd/doc-mpdconf.example.patch#L73), which it what it tries to connect to after the update

**Comment by macerg** [comment]
My mpd server is on my Linux media server. I have SSH tunnels connected using autossh. It's always worked perfectly until yesterday's mpc update. I would like to rollback mpc one version if this can't be fixed. Can you easily facilitate a rollback please? It's not in the apt archives.

**Comment by Grimler91** [comment]
Or you can specify the host with the `--host` option. I guess you want `--host 127.0.0.1`

Use an alias if that is inconvenient: `alias mpc="mpc --host 127.0.0.1 --port 6600"`

**Comment by macerg** [comment]
Yes, just discovered that. mpc -h 127.0.0.1  -p 6600.

It's now complaining about my mpd version.
`warning: MPD 0.21 required`
I'm on the latest version, which is 

`Music Player Daemon 0.23.8 (0.23.8)`

**Comment by macerg** [comment]
I fixed my script. Closing.

**Comment by macerg** [comment]
Fixed my script -h 127.0.0.1
